src/chrome/content/rules/Sparkasse-ansbach.de.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="Sparkasse-ansbach.de">
```

Comment:
Please also add `sparkasse-ansbach.de` as target and redirect rule to `https://www.sparkasse-ansbach.de/`. The issue on `https://sparkasse-ansbach.de` should be documented for future maintenance.